tests/download/test_session.py

```diff
@@ -50,7 +50,7 @@ def test_session(session: SentinelHubSession) -> None:
 def test_token_info(session: SentinelHubSession) -> None:
     info = session.info()
 
-    for key in ["sub", "aud", "jti", "exp", "name", "email", "sid", "org", "did", "aid", "d"]:
+    for key in ["exp", "iat", "jti", "iss", "sub", "typ", "azp", "scope", "account"]:
```

Comment:
Should we only check for what we need/use, i.e. "azp"?

Reply:
done